app/scrapers/scraper.rb

```diff
@@ -26,6 +34,9 @@ def scrape
     end
   end
 
+  # @return [Array] an Array of objects most likely retrieved via an API call.
+  #   Each element will get passed to the scraper template methods in order to
+  #   extract the desired data.
```

Comment:
I don't think that's necessary. It's implied with the return type `Array`, and that we use`map` is more of an implementation detail.